README_vi.md

```diff
@@ -0,0 +1,1647 @@
+# Flight rules cho Git
+
+🌍
+*[English](README.md) ∙ [Español](README_es.md)  ∙  [Русский](README_ru.md) ∙ [简体中文](README_zh-CN.md)∙ [한국어](README_kr.md)*
```

Comment:
Can you add Vietnamese to the other READMEs, too?

Reply:
Ok. I added to  all other readmes